SeaPkg/SeaPkg.dec

```diff
@@ -31,6 +31,7 @@
   HashLibRaw|Include/Library/HashLibRaw.h
   PeCoffLibNegative|Include/Library/PeCoffLibNegative.h
   PeCoffValidationLib|Include/Library/PeCoffValidationLib.h
+  PlatformCpuInformationLib|Include/Library/PlatformCpuInformationLib.h

```

Comment:
This is a fairly generic library class name currently serving a specific use case. We might consider something like "SeaCpuInformationLib". Then if it's less likely to conflict and more obvious in build reports and DSC files that this is something specifically meant for SEA.